lib/iris/analysis/_interpolation.py

```diff
@@ -585,14 +609,22 @@ def __call__(self, sample_points, collapse_scalar=True):
             of the cube will be the number of original cube dimensions minus
             the number of scalar coordinates, if collapse_scalar is True.
 
+        Notes
+        -----
+        .. note::
+
+            If the source cube has lazy data,
+            `chunks <https://docs.dask.org/en/latest/array-chunks.html>`__
+            in the interpolated dimensions will be combined before regridding.
+
```

Comment:
Added note analogous to the one in the [`RectilinearRegridder.__call__` docstring](https://github.com/SciTools/iris/blob/f30980e64d88e25cc2e3efc41eb793619077ed7f/lib/iris/analysis/_regrid.py#L890-L897)